### PR TITLE
Charge transfer equations 

### PR DESCRIPTION
**Pull request by mpmdean**
This adds the equations needed to compute charge-transfer energies as mentioned in #67 .

I'll probably merge this tomorrow if you haven't had a chance to check it.

### CODE REVIEW

File: edrixs/utils.py [hunk 1]
```diff
@@ -275,6 +276,167 @@ def F0F2F4F6_to_UdJH(F0, F2, F4, F6):
...
+    References
+    ----------
+    .. [1] J. Zaanen, G. A. Sawatzky, and J. W. Allen
+           `Phys. Rev. Lett. 55, 418 (1985) <https://doi.org/10.1103/PhysRevLett.55.418>`_
+    .. [2] Maurits Haverkort et al.,
+           `Phys. Rev. B 85, 165113 (2012) <https://doi.org/10.1103/PhysRevB.85.165113>`_
+    .. [3] A. E. Bocquet et al.,
+           `Phys. Rev. B 53, 1161 (1996) <https://doi.org/10.1103/PhysRevB.53.1161>`_
+    """
+    E_imp = n*(U_dd*n + U_dd - 2*Delta)/(2*n + 20)
```
Comment (shenmidelin) [inline edrixs/utils.py:348]: This should be E_L, not E_d

File: edrixs/utils.py [hunk 2]
```diff
@@ -275,6 +276,167 @@ def F0F2F4F6_to_UdJH(F0, F2, F4, F6):
...
+    E_bath = -(U_dd*n**2 + 19*U_dd*n - 20*Delta)/(2*n + 20)
```
Comment (shenmidelin) [inline edrixs/utils.py:349]: This should be E_d

File: edrixs/utils.py [hunk 3]
```diff
@@ -275,6 +276,167 @@ def F0F2F4F6_to_UdJH(F0, F2, F4, F6):
...
+    return E_imp, E_bath
```
Comment (shenmidelin) [inline edrixs/utils.py:350]: The docs string use the names `E_d` and `E_L` for return arguments

File: edrixs/utils.py [hunk 4]
```diff
@@ -275,6 +276,167 @@ def F0F2F4F6_to_UdJH(F0, F2, F4, F6):
...
+    ----------
+    .. [1] J. Zaanen, G. A. Sawatzky, and J. W. Allen
+           `Phys. Rev. Lett. 55, 418 (1985) <https://doi.org/10.1103/PhysRevLett.55.418>`_
+    .. [2] Maurits Haverkort et al.,
+           `Phys. Rev. B 85, 165113 (2012) <https://doi.org/10.1103/PhysRevB.85.165113>`_
+    .. [3] A. E. Bocquet et al.,
+           `Phys. Rev. B 53, 1161 (1996) <https://doi.org/10.1103/PhysRevB.53.1161>`_
+    """
+    E_dc = (10*Delta - n*(31 + n)*U_dd/2 - 90*U_pd) / (16 + n)
+    E_Lc = (10*Delta + (1 + n)*(n*U_dd/2 - (10 + n)*U_pd)) / (16 + n)
```
Comment (shenmidelin) [inline edrixs/utils.py:435]: This should be E_p, not E_Lc

File: edrixs/utils.py [hunk 5]
```diff
@@ -275,6 +276,167 @@ def F0F2F4F6_to_UdJH(F0, F2, F4, F6):
...
+    E_p = ((1 + n)*(n*U_dd/2 + 6*U_pd) - (6 + n)*Delta) / (16 + n)
```
Comment (shenmidelin) [inline edrixs/utils.py:436]: This should be E_Lc

File: edrixs/utils.py (hunk 1 above)
Comment (mpmdean) [reply edrixs/utils.py:348]: changed

File: edrixs/utils.py (hunk 3 above)
Comment (mpmdean) [reply edrixs/utils.py:350]: Yes, I changed notation halfway through writing this.

File: edrixs/utils.py (hunk 1 above)
Comment (shenmidelin) [reply edrixs/utils.py:348]: `E_imp = -(U_dd*n**2 + 19*U_dd*n-20*Delta)/(2*n+20)`
`E_bath=n*(U_dd*n + U_dd - 2*Delta)/(2*n + 20)`

**Pull request merged**